test/rekt/features/trigger/feature.go

```diff
@@ -86,3 +87,43 @@ func TriggerDependencyAnnotation() *feature.Feature {
 
 	return f
 }
+
+func TriggerWithTLSSubscriber() *feature.Feature {
+	f := feature.NewFeatureNamed("Trigger with TLS subscriber")
+
+	brokerName := feature.MakeRandomK8sName("broker")
+	sourceName := feature.MakeRandomK8sName("source")
+	sinkName := feature.MakeRandomK8sName("sink")
+	triggerName := feature.MakeRandomK8sName("trigger")
+
+	eventToSend := cloudeventstest.FullEvent()
+
+	// Install Broker
+	f.Setup("Install Broker", broker.Install(brokerName, broker.WithEnvConfig()...))
+	f.Setup("Broker is ready", broker.IsReady(brokerName))
+	f.Setup("Broker is addressable", broker.IsAddressable(brokerName))
+
+	// Install Sink
+	f.Setup("Install Sink", eventshub.Install(sinkName, eventshub.StartReceiverTLS))
+
+	// Install Trigger
+	f.Setup("Install trigger", func(ctx context.Context, t feature.T) {
+		subscriber := service.AsDestinationRef(sinkName)
+		subscriber.CACerts = eventshub.GetCaCerts(ctx)
+
+		trigger.Install(triggerName, brokerName,
+			trigger.WithSubscriberFromDestination(subscriber))(ctx, t)
+	})
+	f.Setup("Wait for Trigger to become ready", trigger.IsReady(triggerName))
+
+	// Install Source
+	f.Requirement("Install Source", eventshub.Install(
+		sourceName,
+		eventshub.StartSenderToResource(broker.GVR(), brokerName),
+		eventshub.InputEvent(eventToSend),
+	))
+
+	f.Stable("Trigger").Must("deliver events to TLS subscriber", assert.OnStore(sinkName).AtLeast(1))
```

Comment:
We're missing a Match something here

Reply:
I thought as I am sending only one event anyhow, I don't need a matcher :thinking: Updated it anyhow for clarification